public/profile.css

```diff
@@ -0,0 +1,50 @@
+body{
```

Comment:
ここスペース欲しいです！！
他にも見受けられるので確認お願いします！
```suggestion
body {
```